### PR TITLE
Restore travis tests

### PR DESCRIPTION
**Pull request by matanlurey**
My valiant attempt at https://github.com/dart-lang/angular/issues/517.

### CODE REVIEW

**Comment by kevmoo** [comment]
@matanlurey see the travis sadness?

**Comment by natebosch** [comment]
Do we need to set up Travis to install dartium?

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,12 @@ env:
   - PKG=angular_compiler
   - PKG=angular_test
 
-install_dartium: true
+# We don't use the dart_task model yet (mono-repo), so this will have to do.
+before_install:
+  - wget http://gsdview.appspot.com/dart-archive/channels/stable/release/latest/dartium/dartium-linux-x64-release.zip
```
Comment (kevmoo) [inline .travis.yml]: FYI: this is not synced to the version of the SDK for the build

Comment (matanlurey) [inline .travis.yml]: Then I have no idea how to proceed :)

Comment (matanlurey) [inline .travis.yml]: It looks like it isn't working anyway. Still get dartium not found.

Comment (kevmoo) [inline .travis.yml]: I think you need to set up PATH or some such – or create an alias?

**Comment by kevmoo** [comment]
@matanlurey – you're a legend!

**Pull request closed without merging**